app/models/katello/glue/pulp/repo.rb

```diff
@@ -744,6 +744,13 @@ def ostree?
         self.content_type == Repository::OSTREE_TYPE
       end
 
+      def capsule_download_policy
+        if self.is_a?(::Katello::Repository)
+          repo = self.library_instance || self
+          repo.download_policy
+        end
+      end
```

Comment:
I would actually change it to   `if self.yum?` and leave it here.  There are some other methods that are shared between these two classes and live here so i think that is okay.  Unless you are thinking of creating a 2nd module that is shared.